<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajouter_vol</class>
 <widget class="QWidget" name="Ajouter_vol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un vol</string>
  </property>
  <widget class="QLabel" name="lb_date">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date du vol :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_voile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Voile :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_decollage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Décollage :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_distance_cumulee">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Distance cumulée :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_distance">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Distance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_plafond">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Plafond :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_vitesse_moyenne">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vitesse moyenne :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_vitesse_max">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vitesse max :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_vario_max">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vario Max :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_temps_vol">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Temps de vol :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_gain">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gain :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_atterissage">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Atterissage :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_g_max">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>G max :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_date">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_decollage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_atterissage">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_distance">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_distance_cumulee">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_vitesse_max">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_vitesse_moyenne">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_plafond">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_gain">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_temps_vol">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_vario_max">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_g_max">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ajouter_vol">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajouter un vol :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_ajouter_vol">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>111</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ajouter le vol</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_fermer_ajouter_vol">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>171</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fermer la fenêtre</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_Vol_enregistre">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vol enregistré</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
